app/src/main/java/io/github/droidkaigi/confsched2018/util/ext/RecyclerViewExt.kt

```diff
@@ -18,3 +22,9 @@ fun RecyclerView.addOnScrollListener(
         }
     })
 }
+
+fun RecyclerView.addDivider() {
```

Comment:
I think this name is too general because This is specific `6dp` divider.
How about changing name or changing to `setDivider(@DrawableRes drawableResId: Int)`?

Reply:
That's cooler! I will fix it! 🙌